cmake/install.cmake

```diff
@@ -51,7 +59,13 @@ IF(layout_no EQUAL -1)
   MESSAGE(FATAL_ERROR "Invalid installation layout ${INSTALL_LAYOUT}. Please specify one of the following layouts: ${VALID_INSTALL_LAYOUTS}")
 ENDIF()
 
-
+# This has been done before C/C cmake scripts are included
+IF(NOT DEFINED INSTALL_LIB_SUFFIX)
+  SET(INSTALL_LIB_SUFFIX "lib" CACHE STRING "Directory, under which to install libraries, e.g. lib or lib64")
+    IF("${CMAKE_SIZEOF_VOID_P}" EQUAL "8" AND EXISTS "/usr/lib64/")
+      SET(INSTALL_LIB_SUFFIX "lib64")
+    ENDIF()
```

Comment:
The link that Otto provided in ODBC-311, shows
```
-DINSTALL_LIBDIR=lib/$(DEB_HOST_MULTIARCH)/odbc
```
it seems to imply that libraries should go into `/usr/lib/x86_64-linux-gnu/odbc` and not into `/usr/lib64/...`
If it really is the Debian convention, you shouldn't to lib64 here for DEB

Reply:
it's moved from main CMakeLists.txt to install.cmake to set default bintar paths. 
and there are duplicated variables for the same things, yes, but I didn't introduce them